scrapy_zyte_smartproxy/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .middleware import ZyteSmartProxyMiddleware
+
+
+__version__ = '1.7.2'
```

Comment:
Changelog has version v2.0.0 (2021-05-NN). Versions must be same.

Reply:
This is usually done with the `bump2version` tool right before the release, and right after updating the release date in the release notes.